src/pages/About/styles.js

```diff
@@ -102,47 +164,218 @@ const useStyles = createUseStyles( theme => ({
     border: `1px solid ${theme.button1BG}`,
     backgroundColor: theme.button1BG,
     borderRadius: '22.5px',
-    width: '270px',
+    width: '200px',
     height: '45px',
     outline: 'none',
     cursor: 'pointer',
+    margin: 'auto',
+    marginBottom: '40px', 
   },
 /**********     MEDIA-QUERIES     **********/
-  '@media (max-width: 672px)': {
-    projectsContainer: {
-      paddingBottom: '100px'
-    },
-    projectsImg3: {
-      marginTop: '45px',
-    },
-  },
-  '@media (max-width: 935px)': {
-    headingContainer: {
-      margin: '50px auto 0 auto'
-    },
-    heading: {
-      maxWidth: '426px',
-    }
-  },
-  '@media (max-width: 450px)': {
-    heading: {
-      fontSize: '26px',
-      lineHeight: '36px',
-    },
-    text: {
-      fontSize: '20px',
-      lineHeight: '28px',
-    },
-    projectsHeading: {
-      fontSize: '20px',
-      lineHeight: '28px',
-    },
-  },
-  '@media (max-width: 460px)': {
-    getStartedHeading: {
-      fontSize: '30px',
-    },
+'@media (min-width: 675px)': {
+  url: {
+    fontSize: '12px',
+  },
+  headingDiv:{
+    width:'432px',
+  },
+  heading: {
+    fontSize: '28px',
+    lineHeight: '32px',
+  },
+  videoDiv:{
+    backgroundColor: '#F2F2F2',
+    width:'500px',
+    height:'275px',
+    marginTop:'55px',
+    margin:'auto',
+    //paddingTop:'18px',
+    //paddingRight: '10px',
+    //paddingLeft: '10px',
+  },
+  playButtonImg: {
+    height:'85px',
+    width:'85px',
+    //transform: 'translate(0, 40px)',
+    marginLeft:'206px',
+    marginTop:'95px',
+    //margin:'auto',
+  },
+  /**********     TEXT SECTION      **********/
+  textSection: {
+    paddingTop:'56px',
+    paddingBottom:'83px',
+    width: '500px',
+    margin:'auto',
+  },
+  projectsImg1: {
+    marginTop:'40px',
+    width: '273px',
+    height: '259px',
+    //transform: 'translate(0, 40px)',
+  },
+  projectsImg2: {
+    width: '327px',
+    height: '191px',
+    marginTop:'58px',
+    //transform: 'translate(0, 40px)',
+  },
+  projectsImg3: {
+    width: '380px',
+    height: '256px',
+    marginTop:'36px',
+    //transform: 'translate(0, 10px)',
+  },
+  projectsText: {
+    width: '340px',
+    textAlign: 'center',
+    margin:'auto',
+    fontSize: '18px',
+    lineHeight: '24px',
+    fontWeight: '500',
+  },
+  // projectsText2: {
+  //   width: '303px',
+  //   transform: 'translate(0, 42px)',
+  // },
+  projectsText3: {
```

Comment:
It might be worthwhile to find the common elements and created a base project class and add in the variants into it's own file.

Something along these lines.

projects {
textAlign: 'center',
    margin:'auto',
    fontSize: '18px',
    lineHeight: '24px',
    fontWeight: '500',
}

projectWidth340{
width: '340px',
}